stable/fluent-bit/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: fluent-bit
-version: 0.9.0
+version: 0.9.2
```

Comment:
where did 0.9.1 go?

Reply:
sorry, went to another PR that I submitted about 10 minutes before this one.  seems to have disappeared tho. :/

Reply:
found it: https://github.com/helm/charts/pull/7386